daml.yaml

```diff
@@ -14,6 +14,7 @@ parties:
   - Ccp
   - Broker
   - Bank
+  - Issuer
```

Comment:
I think we can get rid of `BtcIssuer`, `UsdtIssuer` and `Custodian` from here as well, those were leftover from the old model's setup script

Reply:
Good points - I'll remove them with the next commit.